pages/server/database/influxdb.mdx

```diff
@@ -11,11 +11,119 @@
 
 To create a server, we suppose that you've already [created and linked an account into the panel](/getting-started).
 
-In that case, go to DBH server and run this command:
+In that case, go to DBH server and,
+
+For a free server:
+
+
+        <DiscordMessages>
+          <DiscordMessage
+            author="Jonfirexbox"
+            avatar="https://avatars.githubusercontent.com/u/66625200?v=4"
+            roleColor="#00cfff"
+            roleName="Client"
+          >
+            DBH!server create influxdb [optional server name]
+          </DiscordMessage>
+          <DiscordMessage
+            author="DanBot Hosting"
+            avatar="https://images-ext-1.discordapp.net/external/9j2_RacvCl8mly7hALccyH0PsM-tbJYm8twQKGf_3jY/https/cdn.discordapp.com/avatars/640161047671603205/718aa5783508125a02e3b244a1bf096e.png"
+            roleColor="#0329d3"
+            bot={true}
+            verified={false}
+          >
+            <DiscordEmbed slot="embeds" color="#00FF00" embed-title="Server Successfully Created">
+						  <DiscordEmbedDescription slot="description">
+							Click Here to Access Your Server
+							</DiscordEmbedDescription>
+							<DiscordEmbedFields slot="fields">
+                <DiscordEmbedField fieldTitle="Status:" inline inline-index="1">
+                  {' '}
+                  Created{' '}
+                </DiscordEmbedField>
+                <DiscordEmbedField fieldTitle="User ID:" inline inline-index="2">
+                  {' '}
+                  126{' '}
+                </DiscordEmbedField>
+                <DiscordEmbedField fieldTitle="Type:" inline inline-index="3">
+                  {' '}
+                  influxdb{' '}
+                </DiscordEmbedField>
+                <DiscordEmbedField fieldTitle="Server Name:" inline inline-index="4">
+                  {' '}
+                  Untitled Server (settings -> server name){' '}
+                </DiscordEmbedField>
+								<DiscordEmbedField fieldTitle="Slots Used:">
+                  {' '}
+                  (1 slot / 6 slots){' '}
```

Comment:
Slots are only for Donator servers, as we have no limit for normal servers, (As that's what it was before) so this will have to be changed.

Reply:
This goes for all of them.

Reply:
Ah ok, ill change it now